examples/listener/example1.tf

```diff
@@ -0,0 +1,50 @@
+module "webapp" {
```

Comment:
Toss these into a single `main.tf` and give the modules different names based on what they do.  "webapp_on_port_8080" or something